target/linux/qualcommax/ipq807x/base-files/etc/init.d/smp_affinity

```diff
@@ -1,36 +1,135 @@
 #!/bin/sh /etc/rc.common
+# shellcheck disable=2155,3019,3043,3057,3060
+# Copyright (c) 2015-2017, The Linux Foundation. All rights reserved.
```

Comment:
Why the addition of copyright, even more so of Linux Foundation one?

Reply:
It was copied over by mistake from an NSS branch for `smp_affinity`.
None of the code from there is used here. It was actually the other way
around.

I used `openwrt/main`:
```
target/linux/qualcommax/ipq807x/base-files/etc/init.d/smp_affinity
```

As the basis for [qosmio/nss-packages](https://github.com/qosmio/nss-packages/blob/NSS-12.4-K6.x/qca-nss-drv/files/qca-nss-drv.init):
```
qca-nss-drv/files/qca-nss-drv.init
```

Just pushed a new commit with the same explanation to correct it. Sorry about that.

Reply:
No worries, just squash these commits as the second one is simply deleting parts of the first one

Reply:
Squashed and pushed!

Reply:
My mistake, hopefully this time everything is OK.